### PR TITLE
Cleanup README

### PR DESCRIPTION
**Pull request by atugushev**
- add installation section
- move low-level examples from README to examples dir

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-[![Coverage](https://codecov.io/gh/list-family/ansq/branch/master/graph/badge.svg)](https://codecov.io/gh/list-family/ansq)<br />
+[![Coverage](https://codecov.io/gh/list-family/ansq/branch/master/graph/badge.svg)](https://codecov.io/gh/list-family/ansq)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/ansq)
 
 Written with native Asyncio NSQ package
 
+## Installation
+
+```commandline
+python -m pip instll ansq
```
Comment (lig) [inline README.md:12]:
```suggestion
python -m pip install ansq
```

File: README.md [hunk 2]
```diff
@@ -28,15 +34,15 @@ Written with native Asyncio NSQ package
...
+## Usage
 
 ### Consumer
 
 A simple consumer reads messages from "example_topic" and prints them to stdout.
 
 ```python
-import ansq
 import asyncio
+import ansq
```
Comment (lig) [inline README.md:45]: This is minor. Usually it rather alphabetically

```python
import ansq
import asyncio
```

or two sections: one for stdlib and another for other lib sources

```python
import asyncio

import ansq
```

**Review by lig — CHANGES_REQUESTED** [review]
A typo and a minor import sorting suggestion

File: README.md (hunk 2 above)
Comment (atugushev) [reply README.md]: Makes sense. Addresssed in 785d526

**Review by lig — APPROVED** [review]
👍🏻

**Pull request merged**